homeassistant/components/rachio/switch.py

```diff
@@ -281,6 +284,15 @@ def device_state_attributes(self) -> dict:
             props[ATTR_ZONE_SHADE] = self._shade_type
         if self._zone_type:
             props[ATTR_ZONE_TYPE] = self._zone_type
+        if self._slope_type:
+            if self._slope_type == "ZERO_THREE":
+                props[ATTR_ZONE_SLOPE] = "Flat"
+            elif self._slope_type == "FOUR_SIX":
+                props[ATTR_ZONE_SLOPE] = "Slight"
+            elif self._slope_type == "SEVEN_TWELVE":
+                props[ATTR_ZONE_SLOPE] = "Moderate"
+            elif self._slope_type == "OVER_TWELVE":
+                props[ATTR_ZONE_SLOPE] = "Steep"
```

Comment:
These should probably be constants

Reply:
```
 if self._slope_type == SLOPE_FLAT:
     props[ATTR_ZONE_SLOPE] = "Flat"
```etc..

Does something like that work? Adding them to `const.py` of course.

Reply:
Works for me.